src/app/meta-campaign/meta-campaign.component.ts

```diff
@@ -188,7 +188,7 @@ export class MetaCampaignComponent implements AfterViewChecked, OnDestroy, OnIni
 
         // Angular scrolls automatically, using setTimeout to delay this scroll to a later task so this gets to
         // set the position the page is left in.
```

Comment:
probably worth changing this comment, "later task" made sense when delay was set at zero, now we're delaying it more than that. Not sure exactly what it is that we're waiting the 200ms for. Maybe something like this?

```suggestion
        // Angular scrolls automatically, using setTimeout to delay this scroll to happen after the automatic scroll
```

Reply:
I'm also not sure exactly, but think it's connected to the router action and possibly to its `scrollPositionRestoration` feature which we have set to true.

Updated the comments to match my understanding better at https://github.com/thebiggive/donate-frontend/pull/1773/commits/1bec8637ce8186fdc606ea32cf9b5ff7bed30946